products/wagtail.md

```diff
@@ -3,128 +3,156 @@ title: Wagtail
     releaseDate: 2020-11-02
+
 -   releaseCycle: "2.10"
     eol: 2021-02-02
     support: 2020-11-02
     lts: false
     latest: "2.10.2"
     latestReleaseDate: 2020-09-25
     releaseDate: 2020-08-11
+
 -   releaseCycle: "2.9"
     eol: 2020-11-02
     support: 2020-08-11
     lts: false
     latest: "2.9.3"
     latestReleaseDate: 2020-07-20
     releaseDate: 2020-05-04
+
 -   releaseCycle: "2.8"
     eol: 2020-08-11
     support: 2020-05-04
     lts: false
     latest: "2.8.2"
     latestReleaseDate: 2020-05-04
     releaseDate: 2020-02-03
+
 -   releaseCycle: "2.7"
     eol: 2021-02-02
     support: 2020-02-03
     lts: true
     latest: "2.7.4"
-
     latestReleaseDate: 2020-07-20
     releaseDate: 2019-11-06
 
 ---
 
-> [Wagtail](https://wagtail.org/) is an open source content management system built on Django, with a strong community and commercial support. It's focused on user experience, and offers precise control for designers and developers.
+> [Wagtail](https://wagtail.org/) is an open source content management system built on Django, with
+> a strong community and commercial support. It's focused on user experience, and offers precise
+> control for designers and developers.
 
-Minor/Feature releases of Wagtail are released every three months. A feature release will usually stop receiving patch release updates when the next feature release comes out. LTS releases receive fixes for security and data-loss related issues. Typically, an LTS release will happen once every four feature releases and receive updates for five feature releases, giving a support period of fifteen months with a three-month overlap. LTS releases will ensure compatibility with at least one [Django LTS release][django-lts].
+Minor/Feature releases of Wagtail are released every three months. A feature release will usually
+stop receiving patch release updates when the next feature release comes out. LTS releases receive
+fixes for security and data-loss related issues. Typically, an LTS release will happen once every
+four feature releases and receive updates for five feature releases, giving a support period of
+fifteen months with a three-month overlap. LTS releases will ensure compatibility with at least
+one [Django LTS release](https://www.djangoproject.com/download/#supported-versions).
 
 The Wagtail team provides [official security support](https://docs.wagtail.org/en/stable/contributing/security.html#supported-versions) for:
 
 * The two most recent Wagtail release series.
 * The latest LTS release.
 
-*[LTS]: Long Term Support
-[django-lts]: https://www.djangoproject.com/download/#supported-versions
-
 ## [Compatible Django / Python versions](https://docs.wagtail.org/en/stable/releases/upgrading.html#compatible-django-python-versions)
 
 Wagtail release | Compatible Django versions | Compatible Python versions
 ----------------|----------------------------|----------------------------
 4               | 3.2, 4.0, 4.1              | 3.7-3.10
```

Comment:
Wagtail 4.1+ is compatible with Python 3.7-3.11, see https://docs.wagtail.org/en/stable/releases/upgrading.html#compatible-django-python-versions

The rest looks good to me :+1:

Reply:
Updated.